.github/workflows/ci.yml

```diff
@@ -221,6 +221,9 @@ jobs:
           - rust: stable
             os: ubuntu-latest
             target: aarch64-unknown-linux-gnu
+          - rust: stable
+            os: ubuntu-latest
+            target: i686-unknown-linux-gnu
```

Comment:
nit, since we are not testing on android maybe change the commit message or add android testing [aarch64-linux-android](https://rust-lang.github.io/rustup-components-history/aarch64-linux-android.html)?

Reply:
I'll change the commit message. I tried adding that platform but ran into other issues that I'll need to handle separately and didn't want to hold up the 32-bit fixes.